geotrellis-sentinelhub/src/main/scala/org/openeo/geotrellissentinelhub/PyramidFactory.scala

```diff
@@ -47,9 +46,9 @@ object PyramidFactory {
   // convenience method for Python client
   // TODO: change name? A 429 response makes it rate-limited anyway.
   def rateLimited(endpoint: String, collectionId: String, datasetId: String, clientId: String, clientSecret: String,
-                  processingOptions: util.Map[String, Any], sampleType: SampleType): PyramidFactory =
+                  processingOptions: util.Map[String, Any], sampleType: SampleType, maxSpatialResolution: CellSize = CellSize(10,10)): PyramidFactory =
```

Comment:
Is this going to be backwards compatible when called from python? I'm not sure if the default arguments work in there.

Reply:
The default arguments should be backwards compatible now. The build fails due to the same two tests that are broken in the develop branch: 
- org.openeo.geotrellissentinelhub.S3ServiceTest.download_stac_dataCanHandleBatchJobRetries
- org.openeo.geotrellissentinelhub.S3ServiceTest.download_stac_data

We should be able to merge without issues.

Reply:
Great!